sdk/communication/communication-sms/review/communication-sms.api.md

```diff
@@ -7,12 +7,12 @@
 import { KeyCredential } from '@azure/core-auth';
 import { OperationOptions } from '@azure/core-http';
 import { PipelineOptions } from '@azure/core-http';
-import { RestResponse } from '@azure/core-http';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public
 export interface SendOptions extends OperationOptions {
-    enableDeliveryReport?: boolean;
+    enableDeliveryReport: boolean;
```

Comment:
Removing the optional here is remove on purpose?

Reply:
I believe this is intentional. The send method has a default option with this value set to fault

Reply:
Let me consult though, considering we can set operations properties in here too, we might have a case where they just want to set the operations params

Reply:
@DominikMe any thoughts? I can foresee an issue when a customer wants to just specify some operations options but has to constantly set the "enableDeliveryReport" flag to true

Reply:
Is there any reason why we stub it and don't just use it or hard-code a GUID?

Reply:
Every property in the options bag must be optional, that's the design of the options bag. It's a good question why the service swagger has a required and a default value, that doesn't feel right.

Reply:
@DominikMe Even though it's required in the service, this should be optional in the SDK though, right?